src/test/java/de/retest/recheck/review/GlobalIgnoreApplierTest.java

```diff
@@ -0,0 +1,46 @@
+package de.retest.recheck.review;
+
+import static de.retest.recheck.review.counter.NopCounter.getInstance;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import de.retest.recheck.ui.Path;
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.descriptors.IdentifyingAttributes;
+import de.retest.recheck.ui.descriptors.MutableAttributes;
+import de.retest.recheck.ui.diff.AttributeDifference;
+
+class GlobalIgnoreApplierTest {
+
+	private static class some {
+	}
+
+	@Test
+	void ignoreElement_should_add_filter() {
+		final GlobalIgnoreApplier cut = GlobalIgnoreApplier.create( getInstance() );
+
+		final Element element = Element.create( "myRetestId", Mockito.mock( Element.class ),
+				IdentifyingAttributes.create( Path.fromString( "/other[1]/some[1]" ), some.class ),
+				new MutableAttributes().immutable() );
```

Comment:
If you do not care about attributes, you could also mock them.

Reply:
This is the common way to use Attributes in tests.